src/test/java/net/lingala/zip4j/MiscZipFileIT.java

```diff
@@ -410,6 +410,19 @@ public void testZipSlipFix() throws Exception {
     }
   }
 
+  @Test
+  public void testUnzipFileZipSlipWithNotNormalizedTarget() throws IOException {
+    ZipFile zipFile = new ZipFile(generatedZipFile, PASSWORD);
+    zipFile.addFiles(FILES_TO_ADD);
+
+    try {
+      zipFile.extractAll(new File(outputFolder.getPath(),
+              ".." + InternalZipConstants.FILE_SEPARATOR + outputFolder.getName()).getAbsolutePath());
+    } catch (ZipException e) {
+      fail("Should not throw an exception");
```

Comment:
This is not necessary. If there is an exception, this method will fail anyway. Please remove the catch block.

Reply:
sure